### DISCUSSION TITLE
Version 3.2.1.1

### DISCUSSION THREAD

**Discussion by Tatsu-syo** [Tatsu-syo/noMeiryoUI#79] [Show and tell]
- 繁体字中国語(台湾)の訳を更新しました。abc0922001様に感謝します。

----

- Improve Simplified Traditional Chinese(Taiwan) translations.Thanks for abc0922001.

<hr /><em>This discussion was created from the release <a href='https://github.com/Tatsu-syo/noMeiryoUI/releases/tag/TAG-3.2.1.1'>Version 3.2.1.1</a>.</em>

**Comment by Tatsu-syo** [comment]
Somewhere, Simplified Chinese(China) and Traditional Chinese(Taiwan) are mixed.
I'm Japanese and I didn't study Simplified Chinese(China) and Traditional Chinese(Taiwan).
I can't review translation changes.
What I should do?

This is why, I released No!! Meiryo UI Version 3.2.1.1.
If anyone has any good ideas, please let me know.

**Reply by abc0922001** [reply]
You can use translation hosting websites such as [Crowdin](https://crowdin.com/page/open-source-project-setup-request) or [Hosted Weblate](https://weblate.org/en/news/archive/new-free-software-projects-hosted-weblate_3/) which offer free hosting services for free, open-source software projects. Crowdin features a translation voting function, where a group of people can vote to select reliable translations.

翻訳ソフトウェアの利用:
あなたは [Crowdin](https://crowdin.com/page/open-source-project-setup-request) や [Hosted Weblate](https://weblate.org/en/news/archive/new-free-software-projects-hosted-weblate_3/) のようなホスティングされたソフトウェアプロジェクトの翻訳ウェブサイトを利用できます。これらは無料で、オープンソースのソフトウェアプロジェクトのための無料のホスティングサービスを提供しています。Crowdin には翻訳の投票機能があり、一群の人々が信頼できる翻訳を選び出すために投票を行うことができます。

**Comment by Tatsu-syo** [comment]
Thanks.
Check limitation.
And we discuss some rules when include translations on [issue no 80](https://github.com/Tatsu-syo/noMeiryoUI/issues/80).
Please check..
